### PR TITLE
updated html parsing to use the same code everywhere

### PR DESCRIPTION
**Pull request by mikekirin**
There was two different html parsers which did the same thing.

### CODE REVIEW

File: src/engraving/tests/compat114_data/fingering-ref.mscx [hunk 1]
```diff
@@ -101,7 +101,7 @@
             <durationType>quarter</durationType>
             <Note>
               <Fingering>
-                <text>1</text>
+                <text><fontsize="8"/>1</text>
```
Comment (cbjeukendrup) [inline src/engraving/tests/compat114_data/fingering-ref.mscx:104]: Given the implementation of `extractFont`, I would expect to see `<font size="8"/>` here, i.e. with space. And I think I've never seen MuseScore write `fontsize` without space.

Comment (mikekirin) [reply src/engraving/tests/compat114_data/fingering-ref.mscx]: fixed

File: src/engraving/infrastructure/htmlparser.cpp [hunk 2]
```diff
@@ -21,17 +21,127 @@
  */
 #include "htmlparser.h"
 
-#ifndef ENGRAVING_NO_QTEXTDOCUMENT
```
Comment (cbjeukendrup) [inline src/engraving/infrastructure/htmlparser.cpp:24]: One last thing: would be good to remove this define from `engraving/CMakeLists.txt`. After that, it's good!

**Review by cbjeukendrup — APPROVED** [review]

**Comment by cbjeukendrup** [comment]
Build failures seem a problem of the master branch

**Pull request merged**